### DISCUSSION TITLE
[FEATURE] Upload/download of analysis results (mount eos / afs?)

### DISCUSSION THREAD

**Discussion by andrzejnovak** [CoffeaTeam/coffea-casa#140] [Ideas]
After producing results (like fit templates/histograms) in coffea-casa, I would need to transfer them elsewhere to run the fit (HiggsCombine does not lend itself to running on casa :D) so an way to export results (or import some inputs) would nice. Perhaps mounting one of the common filesystems like afs or eos, or having a /www folder that could be accessed from the web (wget).

**Comment by oshadura** [comment]
@andrzejnovak thanks for your suggestion! we started to work on testing possibility using xrdcp (ar least) to export results in EOS: https://github.com/CoffeaTeam/coffea-casa/issues/163
